MetadataExtractor/Formats/Exif/ExifReader.cs

```diff
@@ -13,16 +13,14 @@ namespace MetadataExtractor.Formats.Exif
     /// <author>Drew Noakes https://drewnoakes.com</author>
     public sealed class ExifReader : JpegSegmentWithPreambleMetadataReader
     {
-        public const string JpegSegmentPreamble = "Exif\x0\x0";
+        public static ReadOnlySpan<byte> JpegSegmentPreamble => "Exif\x0\x0"u8;
```

Comment:
If we only targeted .NET 7+, this member could be on the interface 😀

Reply:
I work a lot in the VS codebase which still targets .NET Framework. I really enjoy when I can use the newer language features.